elpis/gui/src/components/Transcription/ChooseModel.js

```diff
@@ -68,25 +94,79 @@ class ChooseModel extends Component {
                                 <Link to={urls.gui.engine.index}>
                                     {t("transcription.choose_model.train_new")}
                                 </Link>
+                                <Divider />
+                                {(uploadStatus === "not_started" || uploadStatus === "finished") && 
+                                    <div className="FileUpload">
+                                        <Dropzone
+                                            className="dropzone"
+                                            onDrop={this.onDrop}
+                                            getDataTransferItems={evt => fromEvent(evt)}
+                                        >
+                                            {({getRootProps, getInputProps}) => {
+                                                    return (
+                                                        <div
+                                                            {...getRootProps()}
+                                                            className={classNames("dropzone", {
+                                                                dropzone_active: true,
```

Comment:
@aviraljain99 setting the `dropzone_active` classname to be always true will make the dropzone always orange. We can use the `isDragActive` property to add the `dropzone_active` class dynamically. 

Example:

```
                                            {({getRootProps, getInputProps, isDragActive}) => {
                                                    return (
                                                        <div
                                                            {...getRootProps()}
                                                            className={classNames("dropzone", {
                                                                dropzone_active: isDragActive,
                                                            })}
                                                        >
                                                            <input {...getInputProps()} />
                                                            <p>
                                                                {t("transcription.choose_model.upload_model")}
                                                            </p>
                                                        </div>
                                                    );
                                                }}
```